cypress/e2e/accessibility.cy.ts

```diff
@@ -29,11 +29,11 @@ describe("Accessibility tests in light mode", () => {
     });
 
     it("Checks login page", () => {
-        cy.visit("/clients");
+        cy.visit("/");
         cy.get("button[aria-label='Sign Out Button']").click();
-        cy.url().should("include", "/login");
-        // Forces Page to Render Fully
-        cy.contains("Sign in").should("be.visible");
+        cy.visit("/login");
+
+        cy.get("#login-panel[data-loaded='true']", { timeout: 10000 }).should("exist");
```

Comment:
What does `data-loaded="true"` check for?

Reply:
`useEffect` only runs after hydration, at which point the data-loaded attribute is set to true. With a long timeout, this has the effect of waiting until hydration happens, and means we don't have to wait an arbitrary amount of time and hoping that it'll have hydrated by then.